### PR TITLE
[PLAY-946] React Scaffold

### PR DESCRIPTION
**Pull request by jasperfurniss**
**What does this PR do?** A clear and concise description with your runway ticket url.
https://nitro.powerhrg.com/runway/backlog_items/PLAY-946

**Screenshots:** Screenshots to visualize your addition/change
Screenshot 2023-10-04 at 2 37 30 AM

Screenshot 2023-10-04 at 2 37 40 AM

**How to test?** Steps to confirm the desired behavior:
1. Go to beta/kits and see the beta view.
2. Click on link to toggle to the legacy view.
3. Click on link to toggle back to the beta view.

#### Checklist:
- [X] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [ ] **DEPLOY** I have added the `milano` label to show I'm ready for a review.
- [ ] **TESTS** I have added test coverage to my code.

### CODE REVIEW

File: playbook-website/Gemfile [hunk 1]
```diff
@@ -24,9 +24,6 @@ gem "faker"
 gem "will_paginate"
 gem "sentry-ruby"
 gem "sentry-rails"
-gem 'rack-mini-profiler'
```
Comment (jasperfurniss) [inline playbook-website/Gemfile:27]: These were causing issues locally when testing react, and i needed to remove them.

File: playbook-website/app/javascript/site_styles/_site-style.scss [hunk 2]
```diff
@@ -19,7 +19,6 @@ body {
   &--dark-mode-toggle-desktop {
     display: flex;
     justify-content: flex-end;
-    margin-top: $space_lg;
```
Comment (jasperfurniss) [inline playbook-website/app/javascript/site_styles/_site-style.scss:22]: I removed this in lieu of a bit of a structure change that I did, @nidaqg . See my Flex that wraps some things. Feel free to check if this looks odd in any spots.

File: playbook-website/package.json [hunk 3]
```diff
@@ -24,6 +24,7 @@
     "react": "^17.0.2",
     "react-docgen": "^5.4.3",
     "react-dom": "^17.0.2",
+    "react-router-dom": "^6.16.0",
```
Comment (jasperfurniss) [inline playbook-website/package.json:27]: Added this as a preliminary thinking that the story originally covered some of this. 
We can just leave this in here for now, if that's cool. 
I know we will use them. 

Or if preferable, I can remove until we use them in a follow-up story.

**Comment by jasperfurniss** [comment]
I also put "enhancement" as a label to this PR because of the additions to the Background kit.

File: playbook-website/app/javascript/site_styles/_site-style.scss (hunk 2 above)
Comment (nidaqg) [reply playbook-website/app/javascript/site_styles/_site-style.scss:22]: I like it @jasperfurniss , good enhancement!

**Pull request merged**